scannerazuresc.properties

```
@@ -6,8 +6,13 @@ organizationUrl=https://www.sonarsource.com
 homepageUrl=https://docs.sonarsource.com/sonarcloud/advanced-setup/ci-based-analysis/sonarcloud-extension-for-azure-devops/
 issueTrackerUrl=https://sonarsource.atlassian.net/jira/software/c/projects/SONARAZDO/issues
 sourcesUrl=https://github.com/SonarSource/sonar-scanner-azdo
-archivedVersions=1.44.0,1.44.1,1.44.2,1.45.0,1.46.0,2.0.0,2.0.1
-publicVersions=2.2.0
+archivedVersions=1.44.0,1.44.1,1.44.2,1.45.0,1.46.0,2.0.0,2.0.1,2.2.0
+publicVersions=2.3.1
+
+2.3.1.description=Embed Scanner for .NET 6.2.0.85879
+2.3.1.date=2024-07-30
+2.3.1.changelogUrl=https://sonarsource.atlassian.net/issues/SONARAZDO-395?jql=project%20%3D%2010078%20AND%20fixversion%20IN%20%28sq-6.3.1%2C%20sq-6.3%29
```

Just for the sake of accuracy ;)
```suggestion
2.3.1.changelogUrl=https://sonarsource.atlassian.net/issues/SONARAZDO-395?jql=project%20%3D%2010078%20AND%20fixversion%20IN%20%28sc-2.3.1%2C%20sc-2.3%29
```